backend/fbdev.c

```diff
@@ -56,12 +57,23 @@ static void _twin_fbdev_put_span(twin_coord_t left,
         return;
 
     twin_coord_t width = right - left;
-    off_t off = top * screen->width + left;
-    uint32_t *dest =
-        (uint32_t *) ((uintptr_t) tx->fb_base + (off * sizeof(uint32_t)));
+    off_t off = tx->fb_fix.line_length * top + 4 * left;
```

Comment:
Is the number `4` related to `sizeof(uint32_t)`?
Otherwise can we define a macro to make it looks more meaningful?

Reply:
Since `sizeof(uint32_t)` is always 4, it is not worth having such an expression.